frontends/web/src/locales/en/app.json

```diff
@@ -719,6 +719,10 @@
       "info": "Litecoin is transitioning from addresses starting with '3' to addresses starting with an 'M'.",
       "result": "Legacy format of {{address}}:\n{{legacyAddress}}"
     },
+    "onlyThisCoin": {
+      "description": "To receive other tokens, enable them in the settings.",
+      "warning": "Make sure to only receive {{accountName}} on this address."
```

Comment:
I would maybe make this even more clear, like: "Make sure to only receive {{accountName}} on this address. If you deposit other tokens they might not be accessible until we add that token." ?  

I know they are accessible by exporting the seed to MEW but we should avoid anyone from having to do that..

Reply:
Adjusted the text now, pls check.